jetty-websocket/javax-websocket-client/src/main/java/org/eclipse/jetty/websocket/javax/client/JavaxWebSocketClientContainer.java

```diff
@@ -81,8 +81,14 @@ public JavaxWebSocketClientContainer(Supplier<WebSocketCoreClient> coreClientFac
     public JavaxWebSocketClientContainer(WebSocketCoreClient coreClient)
     {
         super();
-        this.coreClient = coreClient;
-        this.addBean(coreClient);
+
+        if (coreClient != null)
+        {
+            coreClient.setCustomizer(defaultCustomizer);
```

Comment:
If a coreClient is passed in, it is probably because it is already configured in a specific way.  Because customizers are tri-state-additive, then we should not replace the customizer but instead combine them.

So this should be `addSessionCustomizer` and the core client should either maintain a list of combine the customizers some how.

Reply:
the alternative is to avoid passing in the core client... but it feels a bit contrived to pass in just the components needed to construct a core client.